index.html

```diff
@@ -1855,73 +1855,73 @@ <h2>Advanced Concepts</h2>
 this section explores more complex topics about <a>verifiable credentials</a>.
       </p>
 
-      <section class="informative">
-        <h3>Lifecycle Details</h3>
-
-        <p>
 Section <a href="#ecosystem-overview"></a> provided an overview of the
 <a>verifiable credential</a> ecosystem. This section provides more detail about
-how the ecosystem is envisaged to operate, as follows:
+how the ecosystem is envisaged to operate:
         </p>
 
-        <ol>
-          <li>
-The <a>verifier</a> defines its policies for accepting
-<a>verifiable credentials</a> from <a>holders</a> for its supported actions.
-          </li>
+        <figure id="life-cycle-details">
+          <img style="margin: auto; display: block; width: 85%;"
+         src="diagrams/ecosystemdetail.svg" alt="diagram showing how
+         credentials flow from issuer to holder, from holder to holders, and
```

Comment:
`flow from issuer to holder, from holder to holders,`
-> `flow from issuer to holder, and optionally from one holder to another; and how`

Reply:
Good wording. I'll update it.